stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -706,3 +706,16 @@ end
         end
     end
 end
+
+@testset "begin/end in gen_call_with_extracted_types users" begin
+    mktemp() do f, io
+        OLDSTDOUT = stdout
+        redirect_stdout(io)
```

Comment:
You want `redirect_stdout(io) do` here or even better, allow the io to be passed in as a kwarg and use `@code_typed io=devnull`.

Reply:
I just did the quick solution for now, but will make another PR soon that adds that functionality to the `code_X` family; we should be able to choose what io we send those to